Add opportunity list use case (#19)
skuid/SamplePages: Use_Cases/Opportunity_List/README.md

# Opportunity List

Easily view opportunities in a single screen both in chart and table format

> <img  src="opportunity_list.png"  width="300"></img>

Reviewer: ```suggestion
<img  src="opportunity_list.png"></img>
```

Hmm, Github rendering is being particular about this image. From what I've read, removing the width attribute can help, and I'm not sure we need this width to be so narrow. Could we remove it?

Author: Sure

Author: Actually, I made that change and now the image looks real big. It seems all of the other use case images have a set width of 300. The image looked alright on my end - did it not show on yours or did it have some other issue?

Reviewer: Yeah, I'm not seeing it on the PR review screen (even after removing that attribute), but that could just be because it's not merged yet. If you want to put that width back, we can double check after it's merged.

> The table shows all opportunities while the charts and KPIs offer an overview. Search or filter to drill down to the opportunities you want to see, or click on portions of a chart to drill down and see the opportunities in that section. In addition, create a new opportunity using a step-by-step process.

Reviewer: I think this list would be clearer rendered as a list

```suggestion
The table shows all opportunities, while the charts and KPIs offer an overview. 

With this page you can do any of the following: 

- Search or filter to drill down to the opportunities you want to see
- Click on portions of a chart to drill down and see the opportunities in that section
- Create a new opportunity using a step-by-step process
```

Author: Okay, I'll do that. Thanks for the feedback!

## Instructions
- Page API: V2

- Data source: Uses default Salesforce data source

- Design system: [Download this Design System file](CompactOpportunityMgt.designsystem). Use the Import function on the Design System page to add this design system to your org.

> - Page XML: [Copy the XML from this page](Opportunity_List.xml), or save it as an XML file, and upload it as a new page in your Salesforce Org.

Reviewer: ```suggestion
- Page XML: [Copy the XML from this page](Opportunity_List.xml), or save it as an XML file, and upload it as a new page in your Salesforce org.
```

## Notes

> - The chart allows you to drill down into the underlying Opportunity data and make changes if necessary.

Reviewer: Do we need this? Don't we say it in the intro?

Author: Yep, that's a good point, I'll take it out

- Add filters through the Show Filters action, then hide them for more screen real estate while continuing to see the active filters on the table.

> - The KPIs are Skuid UI Only Formula fields - so you do not need to worry about using up any of your roll-summary fields on your Account Object.

Reviewer: ```suggestion
- The KPIs are Skuid UI-Only Formula fields - so you do not need to worry about using up any of your roll-summary fields on your Account object.
```
